### PR TITLE
misc recipe changes

### LINKED ISSUE #102: Update bone recipe
Currently bone makes dye.
image

We should change it to instead make bonemeal.

bone block should be able to make bonemeal also (a larger amount, like 9-10)
image

goblin bone should also make bonemeal
image

### PR DESCRIPTION
**Pull request by Vanikoro**
closes #102

### CODE REVIEW

File: mods/ctg_world/items.lua [hunk 1]
```diff
@@ -7,7 +7,10 @@ minetest.clear_craft({ output = "unified_inventory:bag_medium" })
 minetest.clear_craft({ output = "unified_inventory:bag_large" })
 minetest.clear_craft({ output = "mobs:saddle" })
 minetest.clear_craft({ output = "prefab:boat" })
+minetest.clear_craft({ output = "dye:white" })
```
Comment (SeqSEE) [inline mods/ctg_world/items.lua:10]: This will probably clear all white dye. Have you tested that it doesn't?

File: mods/ctg_world/mod.conf [hunk 2]
```diff
@@ -3,5 +3,7 @@ description = World settings for the Cryptech Test game
 min_minetest_version = 5.4
 author = CryptechServices
 title = Cryptech Test world settings
-depends = default, 3d_armor, spacesuit, vacuum, ctg_jetpack, mesecons, unified_inventory, mobs, beds, farming, aquaz, dungeon_loot, growler, lumpblocks, prefab
+        { "dye:blue","technic:lead_ingot", "dye:red" },
```
Comment (SeqSEE) [inline mods/ctg_world/mod.conf:6]: this looks like a mistake

**Comment by SeqSEE** [comment]
closes #89

File: mods/ctg_world/items.lua (hunk 1 above)
Comment (Vanikoro) [reply mods/ctg_world/items.lua:10]: you were right, so it is fixed now, just bone recipe removed

File: mods/ctg_world/mod.conf [hunk 3]
```diff
@@ -3,5 +3,7 @@ description = World settings for the Cryptech Test game
...
+        { "dye:blue","technic:lead_ingot", "dye:red" },
+depends = default, 3d_armor, spacesuit, vacuum, ctg_jetpack, mesecons, unified_inventory, mobs, beds, farming, aquaz, dungeon_loot, growler, lumpblocks, prefab, technic, vehicle_mash
```
Comment (SeqSEE) [inline mods/ctg_world/mod.conf:8]: why is the stuff about dye here?

File: mods/lightning
Comment (SeqSEE) [inline mods/lightning:1]: this is not up to date

File: mods/vacuum
Comment (SeqSEE) [inline mods/vacuum:1]: This is not up to date

File: mods/3d_armor
Comment (SeqSEE) [inline mods/3d_armor:1]: this is not up to date

File: mods/ctg_airs
Comment (SeqSEE) [inline mods/ctg_airs:1]: this is not up to date

**Review by SeqSEE — APPROVED** [review]

**Pull request merged**